end_to_end/instant_answer.py

```diff
@@ -0,0 +1,70 @@
+import os
+
+from fastapi.testclient import TestClient
+
+os.environ['IS_TEST'] = 'true'
+
+from app import app
+
+client = TestClient(app)
+
+
+def test_paris():
+    response = client.get("http://localhost:5000/v1/instant_answer?q=paris")
+    assert response.status_code == 200
+    assert len(response.json()['data']['result']['places']) > 0
+    assert response.json()['data']['result']['places'][0]['id'] == 'admin:osm:relation:71525'
```

Comment:
I feel like all this could be factored out in a function like `assert_autocomplete(query, doc_id)`